doc/devel/contributing.rst

```diff
@@ -327,7 +327,7 @@ prevent unexpected breaking of code that uses Matplotlib.
 - If possible, usage of an deprecated API should emit a
   `.MatplotlibDeprecationWarning`. There are a number of helper tools for this:
 
-  - Use `.cbook.warn_deprecated()` for general deprecation warnings.
+  - Use ``.cbook.warn_deprecated()`` for general deprecation warnings.
```

Comment:
This was supposed to link, but does not, because it's actually `cbook.deprecation.warn_deprecated()` which is imported up into `cbook`. We always import it from there, but sphinx cannot link to it. For now it's ok to just use inline literal here, but then you should remove the leading dot (Which is sphinx syntax for abbreviated code references).

```suggestion
  - Use ``cbook.warn_deprecated()`` for general deprecation warnings.
```

I want to later move all deprecation related stuff to a module `_api` anyway (#16181).

Reply:
Thanks for explaining it so well!
Done the changes